src/layout/plugin/layout/pluginHeader.tsx

```diff
@@ -17,18 +18,27 @@ interface PluginHeaderProps {
   actions?: React.ReactNode;
   /** Controls if we are on a sub-page. If not passed in we will assume we are at a top level. */
   onBackClick?: MouseEventHandler;
+  search?: {
+    query: string;
+    onChange: (query: string) => void;
+  };
```

Comment:
Is this consistent with how we've done it elsewhere?

Reply:
In `front-client` we have an entirely different `SearchDropdownHeader` but that seemed overkill for that case 🤷